portfolio/src/main/webapp/assets/js/main.js

```diff
@@ -91,3 +91,25 @@ function createListElement(text) {
   liElement.innerText = text;
   return liElement;
 }
+
+function createTestimonialElement(text, author, title){
+    //set up block quote
+    testimonialElement = `
```

Comment:
Consider using dom apis like createElement and appendChild instead of creating raw string which can be bug prone.

Reply:
Can be bug prone and makes the website vulnerable to xss!

Reply:
Just switched over!